src/zui/palette/palette.mdx

```diff
@@ -0,0 +1,580 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/blocks';
+
```

Comment:
I believe that it should be possible to import `theme` from `theme.ts` here and then use the constants from it instead of hardcoded values below.